src/supporting-files/stackbit-banner.html.js

```diff
@@ -53,52 +53,103 @@ module.exports = `<div id="theme-bar" class="theme-bar theme-bar-fixed theme-bar
     body.classList.add("has-theme-bar");
     themebar.classList.remove("theme-bar-hidden");
   }
-  document
-    .querySelector("#remove-theme-bar")
-    .addEventListener("click", function(e) {
-      e.preventDefault();
-      body.classList.remove("has-theme-bar");
-      themebar.classList.add("theme-bar-hidden");
-      sessionStorage.setItem('hideThemeBar', true);
-    });
+
+  const demo = searchParams.get("demo")
+  let themeBarProperties = {}
+
+  // themeBarConfig is loaded from themeBarConfig.js in the <head> of this document. The file is generated seperately in \`stackbit-themes\` repo at build time.
+  if (demo && themeBarConfig) {
+    if (themeBarConfig[demo]) {
+      themeBarProperties = themeBarConfig[demo]
+    }
+  } else {
+    themeBarProperties = {
+      themeBarTitle: searchParams.get("themeBarTitle"),
+      themeBarFork: searchParams.get("themeBarFork"),
+      themeBarNewSite: searchParams.get("themeBarNewSite"),
+      themeBarNewSiteLink: searchParams.get("themeBarNewSiteLink"),
+      themeBarNewSiteTheme: searchParams.get("themeBarNewSiteTheme"),
+      themeBarUserGroup: searchParams.get("themeBarUserGroup")
+    };
+  }
+
+  updateThemeBar = props => {
+    const uiThemeBarTitle = themebar.querySelector(".theme-bar-name");
+    const uiThemeBarForkLink = themebar.querySelector(".theme-bar-fork-link");
+    const uiThemeBarForkName = themebar.querySelector(".theme-bar-fork-name");
+    const uiThemeBarNewSiteLink = themebar.querySelector(".theme-bar-new-site-link");
+    const uiThemeBarNewSiteName = themebar.querySelector(".theme-bar-new-site-name");
+
+    // Update Title in top left
+    if (props.themeBarTitle) {
+      uiThemeBarTitle.textContent = props.themeBarTitle;
+    }
+
+    // Update fork button title or visibility
+    if (props.themeBarFork) {
+      if (props.themeBarFork === "false") {
+        uiThemeBarForkLink.style.display = "none";
+      } else {
+        uiThemeBarForkName.textContent = props.themeBarFork;
+      }
+    }
+
+    // Update create site button title or visibility
+    if (props.themeBarNewSite) {
+      if (props.themeBarNewSite === "false") {
+        uiThemeBarNewSiteLink.style.display = "none";
+      } else {
+        uiThemeBarNewSiteName.textContent = props.themeBarNewSite;
+      }
+    }
+
+    // Update button links and pass through extra query params
+    let newForkLink = new URL(uiThemeBarForkLink.href)
+    let newSiteLink = new URL(uiThemeBarNewSiteLink.href);
+
+    if (props.themeBarNewSiteLink) {
+      newSiteLink = new URL(props.themeBarNewSiteLink);
+    }
+
+    if (props.themeBarForkLink) {
+      newForkLink = new URL(props.themeBarForkLink);
+    }
+
+    if (props.themeBarUserGroup) {
+      newSiteLink.searchParams.set('userGroup', props.themeBarUserGroup)
+    }
+
+    if (props.themeBarNewSiteTheme) {
+      newSiteLink.searchParams.set('theme', props.themeBarNewSiteTheme)
+    }
+    
+    if (props.themeBarNewSiteParams) {
```

Comment:
Amazing! So we can migrate from themeBarUserGroup. Please, ping me after releasing factory with this updated unibit version so I'll update the codebase